src/cpp/session/resources/R.css

```diff
@@ -141,3 +141,112 @@ h3.r-arguments-title + table tr td:first-child {
 nav.topic {
    display: none;
 }
+
```

Comment:
This is for a separate issue, right? https://github.com/rstudio/rstudio/issues/14523 ?

Reply:
Yeah, that's right. I had sworn I worked on that in some PR; apparently this is where I had decided to put it!